test/integration/component/test_VirtualRouter_alerts.py

```diff
@@ -47,6 +47,7 @@ def setUpClass(cls):
             cls).getClsTestClient()
         cls.api_client = cls.testClient.getApiClient()
         cls.services = cls.testClient.getParsedTestDataConfig()
+        cls.hostConfig = cls.config.__dict__["zones"][0].__dict__["pods"][0].__dict__["clusters"][0].__dict__["hosts"][0].__dict__
```

Comment:
Is `cls.config` set here or should it be `cls.services?` Can we add basic check for each dictionary key?

Reply:
This is a configuration, not a test data which we consider as 'services'.

Reply:
Ok, sorry, copy that. Please ignore similar comments, thanks